PDP10/kx10_imp.c

```diff
@@ -900,7 +901,6 @@ t_stat imp_devio(uint32 dev, uint64 *data)
              check_interrupts(uptr);
              break;
         case TYPE_BBN:
-             break;

```

Comment:
You might put a `/* fall through */` comment here (and similarly below) to show that falling through to the next case is intentional.

Reply:
I am not doing a fall through. I am removing a break so that the two cases are handled together. Two cases's following each other do not need a fall through:

             check_interrupts(uptr);  
             break;  
        case TYPE_BBN:  
        case TYPE_WAITS:  
             if (*data & IMP_ODPIEN) {